### DISCUSSION TITLE
rtsp client pulls only track1,so no sound

### DISCUSSION THREAD

**Discussion by zzgundam** [bluenviron/mediamtx#302] [Questions]
## Which version are you using?

v0.15.0

## Question

i use openmcu as rtsp client to pull rtsp stream but i only pulls track1 with video but no sound.can i set rtsp-simple-server to provide only 1 track contains video and sound at the same time?

**Comment by aler9** [comment]
Hello, 

> can i set rtsp-simple-server to provide only 1 track contains video and sound at the same time?

Tracks can contain multiple subtracks only if mpegts is deployed, but i don't think that openmcu supports mpegts. In my opinion, you have to find out why openmcu pulls only one track, by debugging openmcu - the server is probably not responsible for this behavior.

[a comment by an automated account is not shown]
